.gitignore

```diff
@@ -60,3 +60,7 @@ yarn-error.log*
 !.yarn/sdks
 !.yarn/versions
 
+#local file
+.env
+/node_modules
+
```

Comment:
Please remove these changes, we already have .env and /node_modules in this file

```suggestion
```